src/commands/InitCommand.js

```diff
@@ -74,7 +80,8 @@ export default class InitCommand extends Command {
 
     objectAssign(lernaJson, {
       lerna: this.lernaVersion,
-      version: version
+      version: version,
+      owners: owners
```

Comment:
could this just be `owners: this.repository.owners`?

or destructured in above, just `owners`.